From 9880e125f8d959b72b0c71d456fa339323c9b7c3 Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Tue, 25 May 2010 19:00:48 +0800
Subject: [PATCH] ENGR00123807 v4l2 output: fix ic_bypass case for 720p playback

If play a video by using ic bypass mode, and the resolution is
bigger than IC limitation(1024x720), will come out DQUEUE timeout issue.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 drivers/media/video/mxc/output/mxc_v4l2_output.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/media/video/mxc/output/mxc_v4l2_output.c b/drivers/media/video/mxc/output/mxc_v4l2_output.c
index 3cc2e93..66a899f 100644
--- a/drivers/media/video/mxc/output/mxc_v4l2_output.c
+++ b/drivers/media/video/mxc/output/mxc_v4l2_output.c
@@ -1428,6 +1428,7 @@ static int mxc_v4l2out_streamon(vout_data * vout)
 
 		if (vout->ic_bypass) {
 			pr_debug("Bypassing IC\n");
+			vout->pp_split = 0;
 			vout->work_irq = -1;
 			switch (vout->v2f.fmt.pix.pixelformat) {
 			case V4L2_PIX_FMT_YUV420:
-- 
1.5.4.4

